SDALib-CPP/SDAReplay/ReplayMain.cpp

```diff
@@ -1,11 +1,19 @@
 #include "ReplayDriver.h"
 
-int main()
+int main(int p_argc, char **p_path)
 {
-    std::string replayFileName("ReplayData/bbRecording20220330-123618.txt");
+    std::string replayFileName;  // initialize the string
+    for (int i = 1; i < p_argc; i++)
+    {
+        // picks the argument written after --bbfile
+        if (p_path[i] == std::string("--bbfile"))
```

Comment:
can't you use strcomp for this

Reply:
not sure... possibly but we then would need to look at it together. If I use strcomp and then try to test it, nothing happens. if I use this way to compare them, it runs ("trying to connect to speeddreams", same thing as if you would do it manually)

Reply:
nvm, I just didn't know how strcomp worked, I thought it would return a bool, instead it returned an integer. C++ probably gave a silent error and that was why I missed it. already pushed the fix